src/components/Tipseen/__tests__/__snapshots__/tipseen-tests.jest.js.snap

```diff
@@ -151,7 +151,7 @@ Array [
       className="monday-style-dialog-content-component bottom edge-center has-tooltip"
     >
       <div
-        className="monday-style-tooltip monday-style-tooltip-primary padding-size-undefined tipseenWrapper monday-style-tipseen-wrapper tipseenWrapperWithoutCustomWidth monday-style-tipseen-wrapper--without-custom-width"
+        className="tooltip monday-style-tooltip primary monday-style-tooltip-primary paddingSizeUndefined padding-size-undefined tipseenWrapper monday-style-tipseen-wrapper tipseenWrapperWithoutCustomWidth monday-style-tipseen-wrapper--without-custom-width"
```

Comment:
the paddingSizeUndefined is related to the undefined issue we saw in other snapshots or a not related issue?

Reply:
ok I see this is an old bug anyway

Reply:
old one, yes